FrontBundle/Tests/Manager/NodeResponseManagerTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace OpenOrchestra\FrontBundle\Tests\Manager;
+
+use OpenOrchestra\BaseBundle\Tests\AbstractTest\AbstractBaseTestCase;
+use OpenOrchestra\FrontBundle\Manager\NodeResponseManager;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use Phake;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * Test NodeResponseManagerTest
+ */
+class NodeResponseManagerTest extends AbstractBaseTestCase
+{
+    protected $nodeRepository;
+    protected $transverseNode;
+
+    protected $maxAge = array(
```

Comment:
@nicolasThal : I did this as a 'test configuration'. That way it's easy to tweak the datas. But I can put it in the dataProvider if you prefer.